src/BaselineOfMoose/BaselineOfMoose.class.st

```diff
@@ -16,14 +16,9 @@ BaselineOfMoose >> baseline: spec [
 
 	repository := self packageRepositoryURL.
```

Comment:
you are trying to remove this method (IMHO not a good idea)

Reply:
It is the same method than in the superclass.